system-addon/content-src/components/TopSites/_TopSites.scss

```diff
@@ -272,6 +262,23 @@
         }
       }
     }
+
+    &.placeholder {
+      .tile {
+        box-shadow: inset $inner-box-shadow;
+      }
+
+      .screenshot {
+        display: none;
+      }
```

Comment:
Did this need to move from above?

Reply:
The CSS I added wasn't taking effect because of precedence rules. I guess I could've made it even more specific